### DISCUSSION TITLE
Implement option to completely hide UI elements in multiplayer lobbies.

### DISCUSSION THREAD

**Discussion by Ice-man02** [ppy/osu#26473] [Ideas]
The tab/shift+tab works perfectly to hide all of the elements in single player. (It will show the accuracy and percentage of the song completed during breaks, but that's about it) 

I have tried searching many options in the settings to try and hide the small leaderboard that appears at the upper left corner of the screen during multiplayer lobbies. Hiding the score and accuracy is not enough of all the players is not enough. It would be much nicer if I could hide that leaderboard too. 

The older client could perfectly hide all the elements during gameplay (leaderboards showed during the breaks, which was a bit distracting and caused some stress, but still an upgrade from what's currently on lazer). The only thing that remained was a very subtle 1st, 2nd, 3rd counter at the bottom left corner that was so subtle it was not distracting (it would have been appreciated if it could also have been removed, but since lazer is the future one cannot dwell upon it any longer). 

If all the options in the game are unable to hide that specific area, then it becomes slightly misleading.   

However if there is indeed an option hidden somewhere, I would appreciate if I was redirected to it. 
![image](https://github.com/ppy/osu/assets/156289308/4763fd0f-f956-4cf2-bace-f68dea6de1d5)

**Comment by diper111111** [comment]
how to hide this????

**Reply by Walavouchey** [reply]
there is currently no way to hide the leaderboard in multiplayer. this is by design
